pkg/apis/v1alpha1/ipblocksinfo_types.go

```diff
@@ -0,0 +1,40 @@
+/* Copyright © 2024 VMware, Inc. All Rights Reserved.
```

Comment:
Is it necessary to publish in both packages? We are standardizing on `crd.nsx.vmware.com/v1alpha1`, right? Seems no need to add this file?

Reply:
I think we will eventually only keep the vpc/v1alpha1, but currently we need crd.nsx.vmware.com/v1alpha1 and v1alpha1 for the code auto-generation. A clean of those folders will be done at #678

Reply:
That PR is merged, the folders are clear now.